### PR TITLE
Adding Dove IO as Commercial Support

### PR DESCRIPTION
**Pull request by radove**
I would like to add Dove IO as a commercial support resource for the Apache Guacamole project.  

Dove IO is a software engineering and security consulting firm in which has extensive knowledge of implementation and customization of Apache Guacamole. The company believes strongly in open source software and is willing to help other users with deploying Apache Guacamole and ensuring that Apache Guacamole is implemented with a security focused architecture. The company has recently leveraged Apache Guacamole to enable select engineers to remotely access energy field analyzer (Private APN, Cloud Connected IoT Devices) worldwide.

Thanks, Ronnie

### CODE REVIEW

File: _companies/doveio.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+title: Dove IO LLC
+logo: /images/companies/doveio-llc.png
+location: https://www.dove.io
+---
+
+Dove IO is a software engineering and security consulting firm in which has extensive knowledge of implementation and customization of Apache Guacamole. The company believes strongly in open source software and is willing to help other users with deploying Apache Guacamole and ensuring that Apache Guacamole is implemented with a security focused architecture. The company has recently leveraged Apache Guacamole to enable select engineers to remotely access energy field analyzer (Private APN, Cloud Connected IoT Devices) worldwide.
```
Comment (mike-jumper) [inline _companies/doveio.md:7]: Please reduce the content of the blurb to ~50 words.

File: _companies/doveio.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+title: Dove IO LLC
+logo: /images/companies/doveio-llc.png
+location: https://www.dove.io
```
Comment (mike-jumper) [inline _companies/doveio.md:4]: Checking https://www.dove.io, I don't see any current mention of Apache Guacamole support. We don't vet companies listed here, of course, but we do ask that the page linked somehow specifically show that support for Apache Guacamole is available from the company in question.

Is there a different page that should be linked?

File: _companies/doveio.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+---
+title: Dove IO LLC
+logo: /images/companies/doveio-llc.png
```
Comment (mike-jumper) [inline _companies/doveio.md:3]: Please reduce the size of the provided logo to no greater than 144 pixels wide.

**Review by mike-jumper — CHANGES_REQUESTED** [review]
Thanks, @radove! Please see the review comments. For reference, our overall criteria are:

* The company appears to exist (no issue here).
* The provided blurb should be neutral and limited to around 50 words in length.
* The company explicitly lists support for Apache Guacamole as something that's provided.

Things look pretty OK here, but the blurb needs to be shortened and the relevant URL needs to point to a page where availability of Apache Guacamole support in some capacity is explicitly listed.

As all other logos currently present are 144px in size and ~7kb, I'd ask that you also limit the size of your logo - 2300px and 150kb is a bit much. ;)

See: https://lists.apache.org/thread/w413qyxvxkbncqmwbbjl2sx8c202h703

**Comment by radove** [comment]
@mike-jumper Thanks for the feedback. I made changes based on your feedback.  I reduced the size of the logo to 144px, reduced the description a bit, and added a URL to our guacamole information page.  We will continue to evolve the information page as we go forward.  Let me know if any other feedback.

**Review by mike-jumper — APPROVED** [review]
Looks good to me. Thanks!

**Pull request merged**